src/services/routing/uniswapV2.ts

```diff
@@ -1,18 +1,22 @@
-import { Address, encodeFunctionData, erc20Abi } from "viem";
+import { Address, encodeFunctionData, erc20Abi, getAddress } from "viem";
 import { Call, UniswapV2GetAmountsOutArgs } from "../../types";
 import { getUniswapV2Router02Contract } from "../../utils/contractHelpers";
 import UniswapV2Router02Abi from "../../../abis/UniswapV2Router02";
 import { CONTRACT_ADDRESSES } from "../../constants/contracts";
 import { CHAIN_ID } from "../../constants/chain";
 
+const UNISWAP_V2_IGNORE_TOKENS = [
+  getAddress("0x4956b52aE2fF65D74CA2d61207523288e4528f96") // RLP, very low liquidity
```

Comment:
Do we want to explicitly ignore it in case the liquidity improves in the future?

Reply:
see https://github.com/seamless-protocol/rebalance-bot/pull/92#discussion_r2453194170